assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js

```diff
@@ -0,0 +1,242 @@
+	};
+
+	const data = useInViewSelect( ( select ) =>
+		select( MODULES_ANALYTICS ).getReport( reportArgs )
+	);
+
+	const loading = useSelect(
+		( select ) =>
+			! select( MODULES_ANALYTICS ).hasFinishedResolution( 'getReport', [
+				reportArgs,
+			] )
+	);
+
+	const error = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS ).getErrorForSelector( 'getReport', [
+			reportArgs,
+		] )
+	);
+
+	if ( ! analyticsModuleActiveAndConnected ) {
+		return null;
+	}
+
+	if ( error ) {
+		return (
+			<div className="googlesitekit-unique-visitors-chart-widget">
+				<h3>Unique Visitors from Search</h3>
+				<WidgetReportError moduleSlug="analytics" error={ error } />
+			</div>
+		);
+	}
+
+	const googleChartData =
+		extractAnalyticsDashboardData(
+			data || [],
+			0,
+			dateRangeLength,
+			0,
+			1,
+			[ __( 'Unique Visitors', 'google-site-kit' ) ],
+			[ ( x ) => parseFloat( x ).toLocaleString() ]
+		) || [];
+
+	const dates = googleChartData.slice( 1 ).map( ( [ date ] ) => date );
+	const options = {
+		...WPDashboardUniqueVisitorsChartWidget.chartOptions,
+		hAxis: {
+			...WPDashboardUniqueVisitorsChartWidget.chartOptions.hAxis,
+			ticks: dates,
+		},
+		vAxis: {
+			...WPDashboardUniqueVisitorsChartWidget.chartOptions.vAxis,
+		},
+		series: {
+			0: {
+				color: WPDashboardUniqueVisitorsChartWidget.statsColor,
+				targetAxisIndex: 0,
+			},
+			1: {
+				color: WPDashboardUniqueVisitorsChartWidget.statsColor,
+				targetAxisIndex: 0,
+				lineDashStyle: [ 3, 3 ],
+				lineWidth: 1,
+			},
+		},
+	};
+
+	const currentValueIndex = 2;
+	const previousValueIndex = 3;
+	const isZeroChart = ! googleChartData
+		.slice( 1 )
+		.some(
+			( datum ) =>
+				datum[ currentValueIndex ] > 0 ||
+				datum[ previousValueIndex ] > 0
+		);
+
+	if ( isZeroChart ) {
+		options.vAxis.viewWindow.max = 100;
+		options.hAxis.ticks = [ new Date() ];
+	} else {
+		options.vAxis.viewWindow.max = undefined;
+	}
+
+	return (
+		<div className="googlesitekit-unique-visitors-chart-widget">
+			<h3>Unique Visitors from Search</h3>
+			<GoogleChart
+				chartType="LineChart"
+				data={ googleChartData }
+				loadingHeight="270px"
+				loadingWidth="100%"
+				loaded={ ! ( loading || isGatheringData === undefined ) }
+				options={ options }
+				gatheringData={ isGatheringData }
+			/>
+		</div>
+	);
+};
+
+WPDashboardUniqueVisitorsChartWidget.chartOptions = {
+	animation: {
+		startup: true,
+	},
+	chart: {
+		title: 'Unique Visitors from Search',
+	},
+	statsColor: '#5c9271',
+	curveType: 'function',
+	height: 270,
+	width: '100%',
+	chartArea: {
+		height: '80%',
+		left: 30,
+		right: 0,
+	},
+	legend: {
+		position: 'top',
+		textStyle: {
+			color: '#616161',
+			fontSize: 12,
+		},
+	},
+	hAxis: {
+		format: 'M/d/yy',
+		gridlines: {
+			color: '#fff',
+		},
+		textStyle: {
+			color: '#616161',
+			fontSize: 12,
+		},
+	},
+	vAxis: {
+		gridlines: {
+			color: '#eee',
+		},
+		minorGridlines: {
+			color: '#eee',
+		},
+		textStyle: {
```

Comment:
These `textStyle` and `titleTextStyle` options should not be needed if not showing the y-axis labels or title.